app/assets/stylesheets/components/_index.scss

```diff
@@ -8,6 +8,7 @@
 @import "tags";
 @import "offer_card";
 @import "offers_banner";
+@import "booking_card";
 
 a {
```

Comment:
As discussed in the call: Let's see if we can scope that style to a class that is applied only when needed.